app/src/main/java/com/github/se/icebreakrr/model/profile/ProfilesViewModel.kt

```diff
@@ -87,6 +96,17 @@ open class ProfilesViewModel(
     }
   }
 
+  /** Represents the state of a profile picture change. */
+  enum class ProfilePictureState {
+    UNCHANGED,
+    TO_UPLOAD,
+    TO_DELETE
+  }
+
+  fun setPictureChangeState(state: ProfilePictureState) {
+    _pictureChangeState.value = state
+  }
```

Comment:
Even if it seems obvious what those lines do, a description of this function could be nice to keep code uniformity